### ISSUE TITLE
403 Client Error: Forbidden for url: https://api.ring.com/clients_api/session

### ISSUE THREAD

**Issue by windserver** [markuzzi/ring-fhem#6]
Hallo zusammen,

seit gestern ca. 14 Uhr bekomme ich den oben genannten Fehler. Habe den ring connector for FHEM fast 1 Monat ohne Probleme genutzt. Jetzt funktioniert es leider nicht mehr. Nächste Woche läuft mein Probe-Abo bei Ring ab, hat das was damit zu tun.

LG windserver

Hier die gesamte Fehlermeldung:

  File "/opt/fhem/scripts/ring.py", line 44, in <module>
    myring = Ring(ring_user, ring_pass)
  File "/usr/local/lib/python2.7/dist-packages/ring_doorbell/__init__.py", line 54, in __init__
    self._process_cached_session()
  File "/usr/local/lib/python2.7/dist-packages/ring_doorbell/__init__.py", line 87, in _process_cached_session
    self._authenticate()
  File "/usr/local/lib/python2.7/dist-packages/ring_doorbell/__init__.py", line 155, in _authenticate
    req.raise_for_status()
  File "/usr/local/lib/python2.7/dist-packages/requests/models.py", line 940, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 403 Client Error: Forbidden for url: https://api.ring.com/clients_api/session

**Comment by oggy79** [comment]
Nein, Ring hat die API geändert. Hab das selbe mit nem anderen Modul was darauf zugreift. Die Entwickler müssen hier korrigieren, damit die Module wieder laufen
